### ISSUE TITLE
Why move CopyRemoteUrl to submenu?

### ISSUE THREAD

**Issue by peterdeme** [eamodio/vscode-gitlens#1154]
https://github.com/eamodio/vscode-gitlens/commit/e3294a923c915a011db8d6f6cec7a1c7290e6eaf

Why was this done?

The first time I was annoyed with VS Code when they hide git push to a submenu for no reason
![image](https://user-images.githubusercontent.com/19969687/99189408-536e9880-2761-11eb-814e-929a15553275.png)

Now, my favourite extension hides my all time favourite functionality in a submenu as well 😢 
![image](https://user-images.githubusercontent.com/19969687/99189447-7731de80-2761-11eb-943e-05d74f2d9c74.png)

**Comment by eamodio** [comment]
Yeah, I'm sorry and I understand the frustration, but I also got a lot of complaints to GitLens adding lots of menu entries. If there are specific menu options that the community feels strongly about having moved out of the submenus please chime in. And I can look into either always moving it out, or adding an option for it. Although I definitely don't want to add options for more than a couple menu entries -- because the overhead of each one gets hard to deal with.

Hopefully vscode will add the ability to better customize menu contributions.

**Comment by peterdeme** [comment]
Thanks for responding. You can close it if you want.

**Comment by kneepdee** [comment]
Took me some time to find this option after an update too. I did bind it to cmd + shift + C now. Should have done that way earlier as it's such a frequently used feature for me.

**Comment by vanstinator** [comment]
Personally I'd really like to see `CopyRemoteUrl` on the top level menu. I use it almost daily to quickly share code snippets with teammates.

**Comment by eamodio** [comment]
I'm going to leave this open for a bit so that I can gauge the interest in this -- please up vote the top post to say you want this change.

**Comment by sguillia** [comment]
I'd love a setting for this too.

The frustration was increased as it took me two weeks to find out that the feature was still there, hidden in a submenu.  

I thought it was a bug as this following option was checked but nothing appeared.

![Capture du 2021-02-01 17-37-25](https://user-images.githubusercontent.com/15984272/106488707-46398e00-64b4-11eb-9fa5-699ca41ed925.png)

Perhaps there could be a mention of recently moved buttons in that place.

**Comment by peterdeme** [comment]
Meh, closing it.
